src/channel.js

```diff
@@ -414,9 +449,28 @@ class _Channel {
         image.microscopyImages.push(microscopyImage)
       }
     })
+
     if (image.microscopyImages.length === 0) {
-      throw new Error('No VOLUME image provided.')
+      throw new Error('No VOLUME image provided for Optioncal Path ID: ' + 
+      image.blendingInformation.opticalPathIdentifier)
+    }
+
+    image.FrameOfReferenceUID = undefined
+    console.info(image.microscopyImages[0].FrameOfReferenceUID)
+    for (let i = 0; i < image.microscopyImages.length; ++i) {
+      if (image.FrameOfReferenceUID === undefined) {
+        image.FrameOfReferenceUID = image.microscopyImages[i].FrameOfReferenceUID
+      } /* To Do: different microscopyImages metadata of the same channel 
+        (monochorme image) can have different FrameOfReferenceUID, what should we do?
+        for the momnet the FrameOfReferenceUID check is disabled
+        ---------------------------------
+        else if (image.FrameOfReferenceUID !== image.microscopyImages[i].FrameOfReferenceUID) {
+        throw new Error('Optioncal Path ID ' + 
+        image.blendingInformation.opticalPathIdentifier + 
+        ' has volume microscopy images with different FrameOfReferenceUID')
+      }*/
```

Comment:
yes I agree, now is commented otherwise it will not work with datasets server http://34.68.90.36/ studyInstanceUID = '1.3.6.1.4.1.5962.99.1.2103930081.1286074986.1595536829665.3.0 (and I can't show a demo of the new blending UI in slim). I will not apply/close this comment now, but I will apply it as soon as we figure out the the grouping issue in MGHComputationalPathology/slim#22